tests/plugins/editsection/test_editsection.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import print_function, unicode_literals
```

Comment:
Delete this. Only Python 3 is supported.

Reply:
Done.